filclient.go

```diff
@@ -273,6 +276,12 @@ func NewClientWithConfig(cfg *Config) (*FilClient, error) {
 		return nil, err
 	}
 
+	// Create a retrieval event publisher
+	retrievalEventPublisher := rep.New()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
```suggestion
```

Reply:
Ah, yep, I originally had this return an error. Forgot to remove this.

Reply:
Yep, forgot to remove this. Thanks.